modules/landing-page/landing-footer.tsx

```
@@ -0,0 +1,76 @@
+              </ul>
+            </div>
+          ))}
+        </div>
+
+        <div className="mt-16 flex flex-col items-center justify-between border-t border-gray-100 pt-8 lg:flex-row">
+          <p className="text-sm text-gray-600">© 2025 {appConfig.appName}. All rights reserved.</p>
+          <div className="mt-4 flex space-x-8 lg:mt-0">
+            {['Privacy Policy', 'Terms of Service', 'Cookie Policy'].map((link) => (
+              <a
+                key={link}
+                href="#"
```

Biome (1.9.4)

[error] Provide a valid value for the attribute href.

The href attribute should be a valid a URL
Check this thorough explanation to better understand the context.

(lint/a11y/useValidAnchor)